test/asyncio_tests/test_examples.py

```diff
@@ -1503,3 +1504,45 @@ def MongoClient(**kwargs):
         assert isinstance(res["encrypted_unindexed"], Binary)
 
         await client_encryption.close()
+
+
+class MotorAWSLambdaExamples(AsyncIOTestCase):
+    @asyncio_test
+    async def test_shared_client(self):
+        environ.setdefault("MONGODB_URI", "localhost")
+        # Start AWS Lambda Example 1
+        import os
+
+        from motor.motor_asyncio import AsyncIOMotorClient
+
+        client = AsyncIOMotorClient(host=os.environ.get("MONGODB_URI", "localhost"))
+
+        async def async_handler(event, context):
+            return await client.db.command("ping")
+
+        def lambda_handler(event, context):
+            return asyncio.run(async_handler(event, context))
+
+        # End AWS Lambda Example 1
+
+    async def test_IAM_auth(self):
```

Comment:
Since we're skipping the test you can remove the outer `async` to fix the build failure